src/graph/GoExecutor.cpp

```diff
@@ -714,7 +715,9 @@ void GoExecutor::finishExecution() {
             yc.emplace_back(std::move(ptr));
         }
     }
-
+    if (!warningMsg_.empty()) {
+        ectx()->addWarningMsg(std::move(warningMsg_));
```

Comment:
I suggest just copy the content to avoid the warningMsg_  reused after the move operation..

Reply:
Here is the final handle, there is no place to use it again.  but i agree with you.